lib/reader.js

```diff
@@ -20,13 +22,25 @@ const uglify = require('uglify-es');
  * @returns {Promise<string>} - promise resolves to a javascript bundle
  */
 module.exports = async (feeds, options) => {
-    Joi.assert(
-        feeds,
-        Joi.array()
-            .items(feedSchema)
-            .required(),
-        `Expected argument 'feeds' to be an array of feeds.`
+    assert(
+        Array.isArray(feeds),
+        `Expected argument 'feeds' to be an array (of feed arrays).`
     );
+    assert(
+        feeds.length > 0,
+        `Expected argument 'feeds' (array) to be contain at least 1 feed (array).`
+    );
+    assert(
+        feeds[0].length > 0,
+        'Expected at least 1 feed (array) to have more than one entry (object).'
+    );
+    [].concat(...feeds).forEach(feedItem => {
```

Comment:
Doesn't really matter, but: why not just `[...feeds]`?

Reply:
Trying to flatten out feeds and don't think that will work. feeds is an array of arrays and you'd end up with the same thing.

```js
[].concat.apply([], [[1],[2],[3]]) // [1,2,3]
[].concat(...[[1],[2],[3]]) // [1,2,3]
[...[[1],[2],[3]]] // [[1],[2],[3]]
```

Reply:
right, I'm just dumb. sorry!